### ISSUE TITLE
Make Tesy's Tagboard installable as a PWA

### ISSUE THREAD

**Issue by cblanken** [cblanken/tesy-tagboard#65]
Add a [web manifest](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Manifest) and [make the app installable](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Guides/Making_PWAs_installable)

There should be an effort to improve [Progressive Enhancment](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Guides/What_is_a_progressive_web_app#progressive_enhancement) for the app and add testing for various mobile devices.
